.github/workflows/selfdrive_tests.yaml

```diff
@@ -326,6 +332,8 @@ jobs:
       uses: codecov/codecov-action@v3
       with:
         name: ${{ github.job }}-${{ matrix.job }}
+      env:
```

Comment:
doesn't this break forks now?

Reply:
seems to handle empty token fine https://github.com/jnewb1/openpilot/pull/62

```
 [2023-12-13T19:35:00.336Z] ['info'] -> No token specified or token is empty
 ...
 [2023-12-13T19:35:01.985Z] ['info'] {"status":"processing","resultURL":"https://app.codecov.io/github/jnewb1/openpilot/commit/cb7023486f572c65fc041bf65d692f6052e9c00c"}
  ```

Reply:
I thought it'll error on getting the secret if it doesn't exist in the fork repo?

Reply:
I don't have that token in my fork's secrets, seems like it just defaults to empty string if the secret isn't available

Reply:
which was the same case as in this PR's actions since we didn't have the token set when I made the PR